cypress/page/shipping-step.page.ts

```diff
@@ -1,10 +1,10 @@
 class ShippingStepPage {
-  private proceedToCheckoutBtn: string;
   private acceptTerms: string;
+  private proceedToCheckoutBtn: string;
 
   constructor() {
-    this.acceptTerms = "[type='checkbox']";
-    this.proceedToCheckoutBtn = ".cart_navigation button";
+    this.acceptTerms = ".checker input";
+    this.proceedToCheckoutBtn = ".cart_navigation .button";
```

Comment:
see if you can use an attribute to indicate that we are looking for the "Proceed to checkout" button